src/integration-test/resources/db/mod/reports/TrialAttendanceReportITest.sql

```diff
@@ -0,0 +1,203 @@
+
+-- Dummy test data
+insert into juror_mod.judge (id, owner, code, description)
+values (1, '415', '1234', 'Test judge'),
+       (2, '417', '4321', 'Judge Test');
+
+insert into juror_mod.courtroom (id, loc_code, room_number, description)
+values (1, '415', '1', 'large room fits 100 people'),
+       (2, '417', '2', 'large room fits 100 people');
+
+insert into juror_mod.trial (trial_number, loc_code, description, judge, trial_type, trial_start_date, trial_end_date,
+                             anonymous, courtroom)
+values ('T100000002', '415', 'TEST DEFENDANT', 1, 'CIV', current_date, null, false, 1),
+       ('T100000003', '417', 'TEST DEFENDANT', 2, 'CIV', current_date, null, false, 2),
+       ('T100000004', '415', 'TEST DEFENDANT', 1, 'CIV', current_date, null, false, 1),
+       ('T100000005', '415', 'TEST DEFENDANT', 1, 'CIV', current_date, null, false, 1);
+
+
+
+INSERT INTO juror_mod.users (username, "name", active, last_logged_in, "version", team_id, approval_limit, user_type,
+                             email, created_by, updated_by)
+VALUES ('COURT.USER.270.415', 'COURT USER.270.415', true, NULL, 1, NULL, 0.00, 'COURT',
+        'COURT.USER.270.415@justice.gov.uk', NULL, NULL),
+       ('COURT.415', 'COURT 415', true, NULL, 1, NULL, 100000.00, 'COURT', 'COURT.415@justice.gov.uk', NULL, NULL);
+
```

Comment:
If values are null we should remove them from the insert

Reply:
Fixed for this, I didn't mop up all the nulls in the test sql because there's a lot in different places taken from other test sql files.